samples/sampler/.qa-storybook/config.js

```diff
@@ -0,0 +1,23 @@
+import 'core-js';
+import {addParameters} from '@storybook/react';
+import {create} from '@storybook/theming';
+import configure from '../src/configure';
+
+const stories = require.context('../stories/qa', false, /.js$/);
+
+addParameters({
+	options: {
+		theme: create({
+			base: 'dark',
+			brandTitle: 'Agate QA SAMPLER',
```

Comment:
Could you please update 'Agate' to 'AGATE'?

Reply:
sure. pushed it